easy_phi/app.py

```diff
@@ -235,6 +258,22 @@ def main(application):
 
     # TODO: add TCP socket handler to listen for HiSlip requests from VISA
 
+if __name__ == '__main__':
+    # parse command line twice to allow pass configuration file path
+    parse_command_line(final=False)
+    parse_config_file(options.conf_path, final=False)
+    # command-line args override conf file options
+    parse_command_line()
+else:
+    try:
+        parse_config_file(options.conf_path, final=False)
+    except IOError:  # configuration file doesn't exist, use defaults
+        pass
+
+settings = {
```

Comment:
moving it here allows to pass right value of debug and enables autoreload. I.e. this way app automatically restarts if any of imported files was changed.